docs/product_specs/05_stats_h5.md

```diff
@@ -0,0 +1,23 @@
+
+# Statistical Summary (QA HDF5)
+
+This statistical HDF5 file contains a variety of data, including:
+
+* Statistics and metrics about the layers and datasets in the input L1/L2 granule.
+* The computed arrays used to generate the histograms and other plots in the QA report PDF
+* The processing parameters used by QA to generate all of the QA outputs
+    - This includes a copy of the final runconfig file settings
+* A copy of the input L1/L2 product's `identification` group
+    - [!Caution] The QA HDF5's `/science/LSAR/identification/` group 
+      is a copy of the input L1/L2 product's group.
+    - For the QA-specific software version, processing datetime, and 
+      runconfiguration contents, please see the 
+      `/science/LSAR/QA/processing/` group.
+
+The contents of the QA HDF5 file for each product type is provided below.
```

Comment:
Saying "provided below" makes a lot of sense in the context of the .docx file, where these markdowns will be concatenated together, but doesn't make a lot of sense to someone viewing the rendered markdown files, since there won't be anything below this.

How about something like this instead?

```suggestion
The contents of the QA HDF5 file for each product type is provided in the subsequent subsections.
```